concise-mid-tag.cddl

```diff
@@ -158,7 +158,12 @@ version-map = {
 version-type = text .default '0.0.0'
 ; version-scheme is defined in CoSWID
 
-svn-type = int
+eq-svn = int
+min-svn = int
+tagged-eq-svn = #6.1000(eq-svn)
+tagged-min-svn = #6.1001(min-svn)
```

Comment:
`eq-svn` and `min-svn` are throw-away types
```suggestion
tagged-eq-svn = #6.1000(int)
tagged-min-svn = #6.1001(int)
```

Reply:
The way I have represented looks more cleaner  and logically flowing to me, so would prefer to retain the same!

Reply:
Also this is in-line with other types as in line 119..

Reply:
> [..] looks more cleaner and logically flowing

I disagree, let's not add any unnecessary clutter please.